### DISCUSSION TITLE
Yet another complaint about Copilot support

### DISCUSSION THREAD

**Discussion by BillionDollarMistake** [helix-editor/helix#7116] [General] [answered]
I understand your point of view: so far copilots do not comply to any standard protocol, but you just can't turn your head back and pretend copilots do not exist! they have been a revolution and they REALLY save a lot of time for a lot of people, including me.
As soon as you start to implementing it, I will start to fund you on a monthly basis. It's a promise
Can't you find a way to "redirect and standardize" copilots suggestions into a LSP feature?

**Comment by kirawi** [comment, marked as answer]
Please do not create new threads for existing issues. This has already been discussed in depth in https://github.com/helix-editor/helix/discussions/4037 and https://github.com/helix-editor/helix/pull/6865. There are no plans to support non-standard features. It is possible to write a standardized LSP to work with copilot, but nobody who wants copilot support has volunteered to do that.

**Comment by BillionDollarMistake** [comment]
If I made a standardized LSP to work with copilot in Golang or Python, could it be integrated in this project?

**Reply by nxy7** [reply]
If you make standarized LSP I don't think You'd need anyone to integrate it, as You can change config settings to use any LSP you want.

**Reply by gabydd** [reply]
your best bet is probably just to use this pr(though its not going to be merged so more like a fork): https://github.com/helix-editor/helix/pull/6865 if you want to use copilot.
